client/js/components/input.js

```diff
@@ -6,30 +6,28 @@ import Spacer from '../components/spacer'
 const Input = ({ classes, input, label, type, placeholder }) => {
 	return (
 		<div className={classes.child}>
-			<label className={classes.label}>{label}</label>
-			<input className={classes.textInput} type={type} placeholder={placeholder} {...input} />
+			<input
+				className={classes.textInput}
+				type={type}
+				placeholder={placeholder}
+				{...input}
+			/>
 			<Spacer height="10px" />
```

Comment:
It doesn't look like this `Spacer` is doing anything anymore (and whatever it was doing should have probably been done with padding). We can remove it.